.github/workflows/build-triton-wheel.yml

```diff
@@ -46,12 +46,7 @@ jobs:
       matrix:
         py_vers: [ "3.9", "3.10", "3.11", "3.12", "3.13", "3.13t" ]
         device: ["cuda", "rocm", "xpu"]
-        docker-image: ["pytorch/manylinux-builder:cpu", "pytorch/manylinux2_28-builder:cpu"]
-        exclude:
-          - device: "rocm"
-            docker-image: "pytorch/manylinux-builder:cpu"
-          - device: "xpu"
-            docker-image: "pytorch/manylinux2_28-builder:cpu"
+        docker-image: ["pytorch/manylinux2_28-builder:cpu"]
```

Comment:
I believe need to keep:
```
- device: "xpu"
            docker-image: "pytorch/manylinux2_28-builder:cpu"
```

Reply:
I'm curious why it's necessary for them to be on the older version? Looks like later on in the script they install a more modern compiler so wouldn't it make sense for them to just use the compiler in the newer image?

Reply:
I went ahead and removed the lines that install the newer compiler since the manylinux_2_28 image already includes a modern compiler

Reply:
Yes looks like the XPU wheels  are not Manylinux 2.28 yet: https://download.pytorch.org/whl/nightly/xpu/torch 
We should transition XPU builds to Manylinux 2.28 as well.
cc @EikanWang @chuanqi129

Reply:
Maybe @chuanqi129 has an opinion here?

Reply:
Looks like the new image uses the correct devtoolset that they want:

```
[251/385] /opt/rh/gcc-toolset-11/root/usr/bin/c++ -DGTEST_HAS_RTTI=0 -Ithird_party/intel/lib/TritonIntelGPUTransforms
```

From https://github.com/pytorch/pytorch/actions/runs/13577591113/job/37957297977?pr=148129#step:7:979

Reply:
Looks like failures are more related to glibc version:

```
NVVMDialect.cpp:(.text._ZN4mlir4NVVM9BarrierOp5buildERNS_9OpBuilderERNS_14OperationStateENS_9TypeRangeENS_10ValueRangeEN4llvm8ArrayRefINS_14NamedAttributeEEE+0xfa): undefined reference to `std::__glibcxx_assert_fail(char const*, int, char const*, char const*)'
```

Reply:
Okay I re-added xpu builds back for manylinux